intro_to_R_tidyverse/01-intro_to_r.Rmd

```diff
@@ -480,7 +481,11 @@ that is in this list.
 ```{r Double Bracket}
 # This however will give us the data.frame in this spot in the list. 
 variety_list[[1]]
+```
+
+Let's take a look at the structure of this:
```

Comment:
Can't comment on this because it slipped through before: https://github.com/AlexsLemonade/training-modules/pull/47/files#diff-56f021502aa13fdd393a095c2bb9b308R466

Should be `first_item`